doc/user_guide/eds.rst

```diff
@@ -536,7 +536,7 @@ The quantification method needs be specified as either 'CL', 'zeta', or 'cross_s
 If no method is specified, the function will raise an exception.
 
 A list of factors or cross sections should be supplied in the same order as the listed intensities
-(please note that HyperSpy intensities in :py:meth:~._signals.eds.EDSSpectrum.get_lines_intensity
+(please note that HyperSpy intensities in :py:meth:'~._signals.eds.EDSSpectrum.get_lines_intensity'
```

Comment:
Shoot! Ok I'll fix when I get back to my desk after lunch. 

> On 15 Jul 2016, at 10:20, Francisco de la Peña <notifications@github.com> wrote:
> 
> In doc/user_guide/eds.rst:
> 
> > @@ -536,7 +536,7 @@ The quantification method needs be specified as either 'CL', 'zeta', or 'cross_s
> >  If no method is specified, the function will raise an exception.
> >  
> >  A list of factors or cross sections should be supplied in the same order as the listed intensities
> > -(please note that HyperSpy intensities in :py:meth:~._signals.eds.EDSSpectrum.get_lines_intensity
> > +(please note that HyperSpy intensities in :py:meth:'~._signals.eds.EDSSpectrum.get_lines_intensity'
> They should be backquotes
> 
> :py:meth:`~._signals.eds.EDSSpectrum.get_lines_intensity`
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>